docs/data/material/pages.ts

```diff
@@ -221,6 +221,10 @@ const pages: MuiPage[] = [
         pathname: '/material-ui/experimental-api/classname-generator',
         title: 'ClassName generator',
       },
+      {
+        pathname: '/material-ui/experimental-api/css-variables',
+        title: 'CSS variables',
+      },
```

Comment:
I didn't understand why https://mui.com/material-ui/experimental-api/css-variables/ isn't linked in the docs. If it's because https://mui.com/material-ui/experimental-api/css-theme-variables/overview is meant to replace it, then I think there could be a follow-up to

- Delete this
- Copy the API section https://mui.com/material-ui/experimental-api/css-variables/#api
- Copy the live demos https://mui.com//material-ui/experimental-api/css-variables/#customizing-components

Reply:
Right OK. I'm renaming the page to include "legacy" and will merge. The changes here are still a step forward, and the docs are experimental.